Standard/Formats.ecschema.xml

```diff
@@ -22,6 +22,12 @@
             <Unit label="&quot;">u:ARC_SECOND</Unit>
         </Composite>
     </Format>
+	<Format typeName="AngleDM" displayLabel="DegreesMinutes" type="decimal" precision="4" uomSeparator="" formatTraits="keepSingleZero|keepDecimalPoint|showUnitLabel">

```

Comment:
please get spacing correct, other than that this looks good

Reply:
Rectified